packages/explat-client/src/internal/local-storage.ts

```diff
@@ -1,15 +1,16 @@
 /**
  * LocalStorage polyfill from https://gist.github.com/juliocesar/926500
  * length and key methods were added to allow for implementing removeExpiredExperimentAssignments
+ * Altered to match window.localStorage signature exactly
```

Comment:
This reads more like a changelog comment rather than documentation. When implementing an interface, what other non-buggy behavior would there be, other than matching the signature exactly?

Reply:
Ah oops, thanks for catching this !